package.json

```diff
@@ -11,53 +11,63 @@
     "fetch"
   ],
   "packageManager": "pnpm@7.1.0",
-  "main": "./core/dist/index.js",
-  "module": "./core/dist/index.esm.js",
-  "types": "./core/dist/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "typesVersions": {
```

Comment:
It requires TS 4.8 to resolve `types` from `exports` field.

So typesVersion are used here for backward compatibility.